docs/operator-manual/argocd-cm.yaml

```diff
@@ -429,3 +429,8 @@ data:
 
   # application.sync.impersonation.enabled indicates whether the application sync can be decoupled from control plane service account using impersonation.
   application.sync.impersonation.enabled: "false"
+
+  # An optional list of annotation keys to hide in UI on secret resources
+  hide.secret.annotations: |
```

Comment:
I am not a big fan of the property key. I think it should be something like `<group>.<scope>.<feature>`. Something like `resource.sensitive.mask.annotations`. Why?
- It acts on resource, so it goes in the same category as customization, compare options, etc.
- The secret is not configurable, it is hardcoded. So I think we can generalize it to "sensitive" resources.
- We don't hide the annotations, we mask them. I think the configuration should reflect that to the user. (even if the internal functions names use hide)

Reply:
Incorporated this suggestion.